### PR TITLE
Create zip at location where CI is looking for it.

### PR DESCRIPTION
**Pull request by UnityAlex**

### CODE REVIEW

**Review by bholmes — APPROVED** [review]

**Comment by UnityAlex** [comment]
The windows tests are also broken on main. Either I or @mrvoorhe will address those separately.

File: unity/CITools/BuildDriver/CoreCLR.cs [hunk 1]
```diff
@@ -22,7 +22,6 @@ public static void Build(GlobalConfig gConfig)
             "-subset clr+libs",
             $"-a {gConfig.Architecture}",
             $"-c {gConfig.Configuration}",
-            $"-v:{gConfig.DotNetVerbosity}"
```
Comment (mrvoorhe) [inline unity/CITools/BuildDriver/CoreCLR.cs:25]: Why was this removed?

Comment (bholmes) [reply unity/CITools/BuildDriver/CoreCLR.cs:25]: Because we were getting zero feedback from failing builds.
